Sources/SwiftProtobuf/Google_Protobuf_Timestamp+Extensions.swift

```diff
@@ -236,21 +240,28 @@ public extension Google_Protobuf_Timestamp {
         self.init(seconds: s, nanos: n)
     }
 
+    /// Returns a `Google_Protobuf_Timestamp` initialized to the same time as
+    /// the given `Date` object.
```

Comment:
Nit: just say "the given `Date`", since `Date` in Swift isn't really an "object" (if we assume that value types aren't considered "objects").